src/Microsoft.DotNet.Darc/DarcLib/VirtualMonoRepo/VmrCodeflower.cs

```diff
@@ -400,8 +400,16 @@ protected async Task<bool> UpdateDependenciesAndToolset(
                 _fileSystem.DeleteDirectory(commonDir, true);
             }
 
+            // Check if the VMR contains src/arcade/eng/common
+            var arcadeEngCommonDir = sourceRepo / VmrInfo.SourceDirName / "arcade" / Constants.CommonScriptFilesPath;
+            if (!_fileSystem.DirectoryExists(arcadeEngCommonDir))
+            {
+                _logger.LogWarning("VMR does not contain src/arcade/eng/common, skipping eng/common update");
```

Comment:
this should probably be an exception tbh

Reply:
I think warning is better